server/models.js

```diff
@@ -132,13 +132,21 @@ module.exports = {
     const queryInsertPhoto = `
       INSERT INTO photos (answer_id, url)
       VALUES ($1, $2)
+      RETURNING id
     `;
 
     try {
-      const answerResults = await db.client.query(queryInsertAnswer, [question_id, body, name, email])
+      const answerResults = await db.pool.query(queryInsertAnswer, [question_id, body, name, email]);
       const answer_id = answerResults.rows[0].id;
-      const photoResults = await db.client.query(queryInsertPhoto, [answer_id, photos])
-      res.status(201).send('Answer inserted successfully');
+      const photoIds = [];
+
+      (async () => {
+        await Promise.all(photos.map(async (photo) => {
```

Comment:
nice work! such a great way to insert photos.

Reply:
Thanks! ESLint actually flagged that I was doing it inefficiently before and suggested the improvement ([rule: no-await-in-loop](https://eslint.org/docs/rules/no-await-in-loop)).

Eric Lie helped me with the implementation too.